README.cs.md

```diff
@@ -41,7 +41,24 @@ dotnet run --project <path_to_src/Inkluzitron>
 
 ## Produkční deployment
 
-TBD
+Pro produkční nasazení je doporučená možnost použít Docker. V adresáři `src/` se nachází potřebné soubory pro vytvoření kontejneru.
+
+### Lokální sestavení
+
+1) Vstupte do adresáře `src/` (`cd src/`).
+2) Zkopírujte si soubor `environment.template.env` a pojmenujte ho `environment.env` a vyplňte v něm požadované hodnoty.
+3) Spusťte příkaz `docker-compose up`. Bot by se měl automaticky sestavit a spustit.
+
+### DockerHub
+
+Připravte si environment konfiguraci (první dva kroky v kapitole lokální sestavení) a spusťte následující příkazy:
+
+Prvně si vemte `environment.template.env` soubor, přejmenujte jej na `environment.env`. Bude se používat i ve výsledném kontejneru.
```

Comment:
```suggestion
```

Reply:
Proč smazat?

Reply:
Protože to pouze opakuje to co je uvedené v odkazované části, vysloveně o 4 řádky výš, a neuvádí to nic smysluplného